assets/js/util.js

```diff
@@ -11,6 +11,8 @@ var TEXTAREA_AUTO_RESIZE_MINIMUM_WIDTH = 768,
     BACK_TO_TOP_BUTTON_ACTIVATION_HEIGHT = 300,
     THRESHOLD_REQUEST_URL_LENGTH = 2000; // maintain 48 characters buffer for generated parameters
 
+var apyRequestStartTime, apyRequestEndTime, lastNRequests = 10;
```

Comment:
lastNRequests should probably be a constant above. Let's call it `INSTALLATION_NOTIFICATION_REQUESTS_BUFFER_LENGTH`.